### ISSUE TITLE
Add Model by FeatureModelLayer

### ISSUE THREAD

**Issue by x1244** [gwaldron/osgearth#2299]
When adding a model by FeatureModelLayer, set the Style of the model by url() displays the model correctly, but using setModel() method does not display the model. Any ideas?
```
void addFeature(osg::Node* model,osgEarth::Map* themap)
{
    using namespace osg;
    using namespace osgEarth;
    OGRFeatureSource* features = new OGRFeatureSource;
    features->setName("vector-data");
    osgEarth::Point* p = new osgEarth::Point;
    p->set(osg::Vec3d(-90.35, 1.32, 0.0));
    features->setGeometry(p);
    themap->addLayer(features);

    Style style;
    ModelSymbol* ms = style.getOrCreateSymbol<ModelSymbol>();
#if 0
    ms->setModel(model);    //can not display the model
#else
    ms->url()->setLiteral("axes.osgt.500000.scale"); //It's OK
#endif
    StyleSheet* sheet = new StyleSheet;
    sheet->addStyle(style);

    FeatureModelLayer* fml = new FeatureModelLayer;
    fml->setName("models");
    fml->setFeatureSource(features);
    fml->setStyleSheet(sheet);
    themap->addLayer(fml);
}
```
There is a similar issues about setURL() and setModel() #2298

**Comment by gwaldron** [comment]
I pushed a fix for this issue. Let me know if you have any problems. Thanks :)

**Comment by x1244** [comment]
It was fixed by [61a9555](https://github.com/gwaldron/osgearth/commit/61a9555d58f8d724c8ab5f2cc705a6b5b3132499).A commit follow https://github.com/gwaldron/osgearth/commit/01e8738fa031eadb223bda609ffae1b28b2f7c77.
